app/avo/actions/yank_rubygems_for_user.rb

```diff
@@ -0,0 +1,26 @@
+class YankRubygemsForUser < BaseAction
+  self.name = "Yank all Rubygems"
+  self.visible = lambda {
+    current_user.team_member?("rubygems-org") &&
+    view == :show &&
+    resource.model.rubygems.present?
+  }
+
+  self.message = lambda {
+    "Are you sure you would like to yank all rubygems for user #{record.handle} with #{record.email}?"
+  }
+
+  self.confirm_button_label = "Yank all Rubygems"
+
+  class ActionHandler < ActionHandler
+    def handle_model(user)
+      rubygems = user.rubygems
+
+      User.transaction do
```

Comment:
handle_model is already called within a transaction so I don't think we need the nested transaction